scripts/genesis.go

```diff
@@ -0,0 +1,82 @@
+//go:build ignore
+// +build ignore
+
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/skip-mev/slinky/cmd/constants"
+	mmtypes "github.com/skip-mev/slinky/x/marketmap/types"
+)
+
+var (
+	useCore    = flag.Bool("use-core", false, "use core markets")
```

Comment:
we only have raydium / core? Ig for dydx / research-markets we're j pointing at the external sources (dydx-nodes, research-json)

Reply:
yea thats the plan